### PR TITLE
Supporting bg images for Sections

### PR DESCRIPTION
**Pull request by mkbansal1**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/en/drafts/mohit/test-page
- After: https://section-fix--world-bank--aemsites.hlx.live/en/drafts/mohit/test-page

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -209,6 +267,8 @@ async function loadLazy(doc) {
   const element = hash ? doc.getElementById(hash.substring(1)) : false;
   if (hash && element) element.scrollIntoView();
 
+  decorateSectionImages(doc);
```
Comment (thomsebastin) [inline scripts/scripts.js:270]: is it possible to use `createOptimisedPicture` from aem.js for this? @mkbansal1

Comment (mkbansal1) [reply scripts/scripts.js:270]: That method use single image only, we have 3 images, one for each viewport

File: scripts/scripts.js [hunk 2]
```diff
@@ -197,6 +197,64 @@ export const fetchLangDatabyFileName = async (fileName) => {
...
+      sourceDesktop.srcset = `${pathname}?width=1920&format=webply&optimize=medium`;
+      sourceDesktop.media = '(min-width: 1024px)';
+      picture.appendChild(sourceDesktop);
+      defaultImgUrl = pathname;
+    }
+
+    if (sectionTabImg) {
+      const sourceTab = document.createElement('source');
+      const { pathname } = new URL(sectionTabImg, window.location.href);
+      sourceTab.type = 'image/webp';
```
Comment (rrrragas) [inline scripts/scripts.js:227]: can these lines of code to be moved to common function with source as one of the argument for differentiation

File: scripts/scripts.js [hunk 3]
```diff
@@ -197,6 +197,64 @@ export const fetchLangDatabyFileName = async (fileName) => {
...
+    }
+
+    if (sectionMobImg) {
+      const sourceMobile = document.createElement('source');
+      const { pathname } = new URL(sectionMobImg, window.location.href);
+      sourceMobile.type = 'image/webp';
+      sourceMobile.srcset = `${pathname}?width=600&format=webply&optimize=medium`;
+      picture.appendChild(sourceMobile);
+      defaultImgUrl = pathname;
+    }
```
Comment (thomsebastin) [inline scripts/scripts.js:241]: Just one small refactor here. Looks good. @mkbansal1 
```js
// Helper function to create a <source> element
const createSource = (src, width, mediaQuery) => {
  if (!src) return;
  const { pathname } = new URL(src, window.location.href);
  const source = document.createElement('source');
  source.type = 'image/webp';
  source.srcset = `${pathname}?width=${width}&format=webply&optimize=medium`;
  source.media = mediaQuery;
  picture.appendChild(source);
  defaultImgUrl = pathname;
};

// Create source elements for different screen sizes
createSource(image, 1920, '(min-width: 1024px)');
createSource(tabImage, 1024, '(min-width: 768px)');
createSource(mobImage, 600, '(max-width: 767px)');

File: scripts/scripts.js (hunk 2 above)
Comment (mkbansal1) [reply scripts/scripts.js]: done

File: scripts/scripts.js (hunk 3 above)
Comment (mkbansal1) [reply scripts/scripts.js]: Done

**Review by thomsebastin — APPROVED** [review]

**Pull request merged**